### PR TITLE
Fix TIME format parsing.

### PR DESCRIPTION
**Pull request by gz**
Missing a % for the after-second part.

### CODE REVIEW

File: sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs [hunk 1]
```diff
@@ -602,7 +597,7 @@ impl Serialize for Time {
         S: Serializer,
     {
         let time = self.to_time();
-        serializer.serialize_str(&time.format("%H:%M:%S.9f").to_string())
+        serializer.serialize_str(&time.format("%H:%M:%S.%9f").to_string())
```
Comment (mihaibudiu) [inline sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: how come the tests passed?

Comment (gz) [reply sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: No idea, maybe you are not using this type?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: maybe you can fix the documentation too?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: I think this fix is wrong, it should be %.9f

Comment (gz) [reply sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: Well we can fix it in your PR hopefully: https://github.com/feldera/dbsp/pull/440

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs:600]: https://docs.rs/chrono/latest/chrono/naive/struct.NaiveTime.html#method.format

**Pull request merged**